nextroute/common/duration.go

```diff
@@ -0,0 +1,69 @@
+package common
+
+import (
+	"fmt"
+	"time"
+)
+
+// DurationUnit is the unit of duration.
+type DurationUnit int
```

Comment:
Why is all of this necessary and we don't simply use Go built-in's `time.Duration`?